app/src/main/res/values/strings.xml

```diff
@@ -79,174 +79,174 @@
     <string name="grid_size">Grid size</string>
     <string name="search_on_s">Search on %s</string>
     <string name="delete_manga">Delete manga</string>
-    <string name="text_delete_local_manga">Do you really want to delete \"%s\" from your phone\'s local storage? \nThis operation cannot be undone.</string>
+    <string name="text_delete_local_manga">Delete \"%s\" from device permanently?</string>
```

Comment:
Too short messages, it can be more informative

Reply:
It may not be a phone, but is always a device.
"local storage" seems technical for no reason.